rustfmt.toml

```diff
@@ -0,0 +1 @@
+tab_spaces = 4
```

Comment:
Not sure if this file needs to be versioned, right? @jpraynaud

Reply:
It looks like the [default value](https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#tab_spaces) for `tab_spaces` is already `4`, so this file can be removed from the PR :+1:

Reply:
done, thanks!